core/node/options.go

```diff
@@ -82,6 +82,8 @@ type Options struct {
 
 	SwarmDialerDNSHeadstart time.Duration
 	PilvytisAddress         string
+
+	NATStatusTrackerV2 OptionsNATStatusTrackerV2
```

Comment:
I think you've missed the part where you set the value in `GetOptions`
Also you probably need to set it for mobile somewhere?

Reply:
AFAIK it's useless for mobile/consumers